From a766045f65f934df3b5f1aa63bc86fbb3e003a09 Mon Sep 17 00:00:00 2001
From: Anu Aliyas <anu.aliyas@qt.io>
Date: Wed, 17 Apr 2024 12:35:58 +0200
Subject: Add missing dependencies to avoid compilation errors

Missing dependencies are added to resolve the dependencies issues
encountered when compiling with Ninja 1.12.0

Fixes:QTBUG-124375
Change-Id: I0f23ef6c60a6f01ee11bffd46121fd7f98dc022c
Reviewed-on: https://codereview.qt-project.org/c/qt/qtwebengine-chromium/+/555586
Reviewed-by: Michal Klocek <michal.klocek@qt.io>
---
 chromium/content/public/browser/BUILD.gn                         | 1 +
 chromium/extensions/browser/api/declarative_net_request/BUILD.gn | 1 +
 2 files changed, 2 insertions(+)

diff --git a/src/3rdparty/chromium/content/public/browser/BUILD.gn b/src/3rdparty/chromium/content/public/browser/BUILD.gn
index d38fa8d303a..c5813115e88 100644
--- a/src/3rdparty/chromium/content/public/browser/BUILD.gn
+++ b/src/3rdparty/chromium/content/public/browser/BUILD.gn
@@ -539,6 +539,7 @@ jumbo_source_set("browser_sources") {
     "//cc",
     "//components/services/storage/public/cpp",
     "//components/viz/host",
+    "//components/spellcheck:buildflags",
     "//content/browser",  # Must not be public_deps!
     "//device/fido",
     "//gpu",
diff --git a/src/3rdparty/chromium/extensions/browser/api/declarative_net_request/BUILD.gn b/src/3rdparty/chromium/extensions/browser/api/declarative_net_request/BUILD.gn
index 1fc492f5a0c..13a266e22f1 100644
--- a/src/3rdparty/chromium/extensions/browser/api/declarative_net_request/BUILD.gn
+++ b/src/3rdparty/chromium/extensions/browser/api/declarative_net_request/BUILD.gn
@@ -23,6 +23,7 @@ source_set("declarative_net_request") {
     "//extensions/common",
     "//extensions/common/api",
     "//services/preferences/public/cpp",
+    "//components/web_cache/browser",
   ]
 
   public_deps = [ "//extensions/browser:browser_sources" ]
-- 
2.45.0

